flatdata-generator/flatdata/generator/templates/rust/archive.jinja2

```diff
@@ -109,15 +109,13 @@ impl ::std::fmt::Debug for {{archive.name}} {
 }
 
 impl {{archive.name}} {
+    #[allow(unused_imports, unused_variables)]
```

Comment:
This is a bit less granular, though, isn't it?

What's the problem with annotating exactly the imports/variable that might be unused?

Reply:
It did not work in the first place:

```
683 |         #[allow(unused_variables)]
     |         ^^^^^^^^^^^^^^^^^^^^^^^^^^ help: if you just forgot a `!`, use: `#![allow(unused_var
```

There is no way to annotated a single import as maybe being unused.

Reply:
But annotating a single variable is possible, should we do that?